src/lib.rs

```diff
@@ -595,6 +615,114 @@ impl Bump {
         }
     }
 
+    /// Allocates a new slice of size `len` slice into this `Bump` and return an
+    /// exclusive reference to the copy.
+    ///
+    /// All elements of the slice are initialized to `value`.
+    ///
+    /// ## Panics
+    ///
+    /// Panics if reserving space for the slice would cause an overflow.
+    ///
+    /// ## Example
+    ///
+    /// ```
+    /// let bump = bumpalo::Bump::new();
+    /// let x = bump.alloc_slice_fill_copy(5, 42);
+    /// assert_eq!(x, &[42, 42, 42, 42, 42]);
+    /// ```
+    #[inline(always)]
+    #[allow(clippy::mut_from_ref)]
+    pub fn alloc_slice_fill_copy<T: Copy>(&self, len: usize, value: T) -> &mut [T] {
```

Comment:
Hey @TethysSvensson thanks for implementing the feature I requested, that was quick! One remark: I messed up the order of arguments in the `vec![]` macro in my request (as I always do, it seems impossible to remember!) - the correct order is `vec![value, len]`. So I think here we should use the same order fo consistency.

Reply:
You are right, that makes more sense.